src/main/java/org/twilmes/sql/gremlin/processor/SingleQueryExecutor.java

```diff
@@ -70,9 +75,13 @@
     private static final String OUT_E_LABEL_KEY = "__out_e_label__";
     private static final String MAP_KEY = "__map__";
     private static final String ID_KEY = "__id__";
+    private static final long RESULT_LIMIT = 10000;
+    private static final int DEFAULT_PAGE_SIZE = 1000;
+    private static int pageSize = DEFAULT_PAGE_SIZE;
```

Comment:
I don't think pageSize should be static.

Reply:
I had to make it static because I was accessing/updating it from the SqlGemlinQueryResult class which is static, but I guess it now depends on what how we want to implement the fetch size later.